specification/schemas/BaseCustoms.json

```diff
@@ -5,16 +5,7 @@
     "non_delivery",
     "incoterm"
   ],
-  "additionalProperties": false,
```

Comment:
You can probably keep ` "additionalProperties": false` in here if you add `content_type` as an enum with all the possibilities.